next.config.mjs

```diff
@@ -1,5 +1,6 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  output: "standalone", // TODO: debug this further
```

Comment:
This comment should be removed or made more specific how manual copying is needed within package.json.